platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationBuilderImpl.java

```diff
@@ -88,6 +98,9 @@ public Configuration build() {
             if (initialized.get()) {
                 throw new IllegalStateException("Configuration already initialized");
             }
+            if (!properties.isEmpty()) {
+                withSource(new SimpleConfigSource(properties).withOrdinal(PLATFORM_BUILDER_ORDINAL));
```

Comment:
Ahh, no I see where `PLATFORM_BUILDER_ORDINAL` is used. Please rename it and place it here :)

Reply:
Since we want to reference this value in the javadocs on the public interface `ConfigurationBuilder`, I moved this constant to `ConfigurationBuilder.java` and renamed it to `CUSTOM_PROPERTY_ORDINAL`.